visualization/src/pcl_plotter.cpp

```diff
@@ -64,9 +64,9 @@
 
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 pcl::visualization::PCLPlotter::PCLPlotter (char const *name)
+  : view_ (vtkSmartPointer<vtkContextView>::New ())
 {
   //constructing
-  view_ = vtkSmartPointer<vtkContextView>::New ();
```

Comment:
Why is it only the vtkContextView that is necessary to change here?^
And not chart_, color_series_, exit_loop_timer_ nor exit_callback_?

Reply:
Not sure. Did you run cppcheck for those files?

Reply:
No I didn't. Just trying to reason why it was only one of them, when they all are created as vtkSmartPointers.

Reply:
Ok, i  added them as well.